executor/runcexecutor/executor_linux.go

```diff
@@ -44,13 +44,10 @@ func (w *runcExecutor) exec(ctx context.Context, id, bundle string, specsProcess
 type runcCall func(ctx context.Context, started chan<- int, io runc.IO) error
 
 func (w *runcExecutor) callWithIO(ctx context.Context, id, bundle string, process executor.ProcessInfo, started func(), call runcCall) error {
-	runcProcess := &startingProcess{
-		ready: make(chan struct{}),
-	}
+	runcProcess, runcCtx, cancel := runcProcessHandle(ctx, id)
 	defer runcProcess.Release()
 
-	var eg errgroup.Group
-	egCtx, cancel := context.WithCancel(ctx)
+	eg, egCtx := errgroup.WithContext(runcCtx)
```

Comment:
on line 64 (can't leave comment there), is `ctx` correct?

I think we should be able to pretty much just use `ctx` var now, without the need for `runcCtx` and `egCtx` variables.

Reply:
Good catch on line 64.  I have updated to use ctx consistently now.  I also had to copy over the logger from the request context to preserve that for logging in the errgroups.